components/pipedrive/actions/add-activity/add-activity.mjs

```diff
@@ -1,133 +1,154 @@
+  version: "0.1.3",
   type: "action",
   props: {
-    pipedrive: {
-      type: "app",
-      app: "pipedrive",
-    },
-    companydomain: {
-      type: "string",
-      description: "Your company name as registered in Pipedrive, which becomes part of Pipedrive API base url.",
-    },
+    pipedriveApp,
     subject: {
       type: "string",
+      label: "Subject",
       description: "Subject of the activity",
     },
     done: {
       type: "integer",
+      label: "Done",
       description: "Whether the activity is done or not. 0 = Not done, 1 = Done",
       optional: true,
     },
     type: {
       type: "string",
+      label: "Type",
       description: "Type of the activity. This is in correlation with the key_string parameter of ActivityTypes.",
     },
-    due_date: {
+    dueDate: {
       type: "string",
+      label: "Due Date",
       description: "Due date of the activity. Format: YYYY-MM-DD",
       optional: true,
     },
-    due_time: {
+    dueTime: {
       type: "string",
+      label: "Due Time",
       description: "Due time of the activity in UTC. Format: HH:MM",
       optional: true,
     },
     duration: {
       type: "string",
+      label: "Duration",
       description: "Duration of the activity. Format: HH:MM",
       optional: true,
     },
-    user_id: {
-      type: "integer",
+    userId: {
+      propDefinition: [
+        pipedriveApp,
+        "userId",
+      ],
       description: "ID of the user whom the activity will be assigned to. If omitted, the activity will be assigned to the authorized user.",
-      optional: true,
     },
-    deal_id: {
+    dealId: {
       type: "string",
+      label: "Deal ID",
       description: "ID of the deal this activity will be associated with",
       optional: true,
     },
-    person_id: {
-      type: "integer",
+    personId: {
+      propDefinition: [
+        pipedriveApp,
+        "personId",
+      ],
       description: "ID of the person this activity will be associated with",
-      optional: true,
     },
     participants: {
       type: "any",
+      label: "Participants",
       description: "List of multiple persons (participants) this activity will be associated with. If omitted, single participant from person_id field is used. It requires a structure as follows: [{\"person_id\":1,\"primary_flag\":true}]",
       optional: true,
     },
-    org_id: {
-      type: "integer",
+    organizationId: {
+      propDefinition: [
+        pipedriveApp,
+        "organizationId",
+      ],
```

Comment:
`organizationId` should use async options
https://developers.pipedrive.com/docs/api/v1/Organizations#getOrganizations